### PR TITLE
Portfolio popup window

### PR DESCRIPTION
**Pull request by iBel251**
- Add popup with project details for each card button
- Applied for both mobile and desktop
- Dynamically rendered all the works section at page load.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -118,80 +118,37 @@ <h1>Multi-Post Stories</h1>
...
+        </ul>
+
+        <div class="modal-body">
+          <div class="modal-img-container">
+            <img src="images/mobile-modal.png" class="modal-img" alt="project 1 modal" >
+          </div>
+          <div class="modal-info">
+            <div class="modal-text">
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+            </div>
```
Comment (micheaol) [inline index.html]: Please ensure that the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). Please remove the hard-cod in your `html file`.

File: script.js [hunk 2]
```diff
@@ -15,3 +15,117 @@ for (let i = 0; i < 3; i += 1) {
...
+
+const projects = [
+  {
+    id: 1,
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    imageUrl: 'gymfit.png',
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    demoLink: '#',
```
Comment (micheaol) [inline script.js:27]: Please ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins).

Your Implementation | Figma Design
:---:|:---:
image | image

##Mobile version:
Your Implementation | Figma Design
:---:|:---:
image | image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @iBel251 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Lordkaito — APPROVED** [review]
Hi <STUDENT_MENTION>, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] [OPTIONAL] It would be awesome if you can get rid of the whitespace on your mobile version. To see this, go to your browser, enter the mobile version of your project and then change the width of your screen to 400px or more. You will realize that the bigger your screen is, the bigger that whitespace.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**